integrations/cesium/README.md

```diff
@@ -32,4 +32,27 @@ Toolsets:
   - LiDAR of Melbourne available as [Open Data](https://data.melbourne.vic.gov.au/City-Council/City-of-Melbourne-3D-Point-Cloud-2018/).
   - LiDAR of Vancouver available as [Open Data](https://opendata.vancouver.ca/pages/home/).
 
+## How this Integration works
+<!-- TODO: Explain the design steps used in this sample -->
+
+1. Start with the basic sandbox Cesium scene
+1. Add a Screen Space quad based on the [custom primitive example](https://github.com/cesiumlab/cesium-custom-primitive)
+  a. In the shader, add 2 texture/sampler sets; one will be the colour buffer, one will be the depth buffer
+  a. Set up the custom primitive to upload a random colour every frame to test upload support
+1. Add the VaultSDKJS Module
+  a. Add login box to the sceen
+  a. Identify the geospatial zone used in Cesium (default is EPSG:4978)
+1. In the custom primitive:
+  a. Render the UDS files in the Update function
+  a. Upload the colour and depth buffer
+  a. Resize the textures when the screen changes (update both the GPU buffer and the VaultSDKJS buffers)
```

Comment:
`a.` doesn't work how I hoped.

Reply:
Based on how it works in comments, three spaces and `1.` will use `i.` and `ii.` instead.